src/main/java/de/helfenkannjeder/come2help/server/rest/AbilitiesController.java

```diff
@@ -57,7 +63,7 @@ public AbilityDto updateAbility(@NotNull @PathVariable Long id, @Valid @RequestB
 
     @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    public void deleteAbility(@NotNull @PathVariable Long id) {
+    public void deleteAbility(@NotNull @PathVariable(value = "id") Long id) {
```

Comment:
It is just for jsondoc. Without the api documentation is not completed and the "playground" does not show all parameters properly.